doc/protocol.rst

```diff
@@ -1006,21 +1006,20 @@ remote playlists (absolute URI with a supported scheme).
 
 .. _command_save:
 
-:command:`save {NAME}`
+:command:`save {NAME} [MODE]`
     Saves the queue to
     `NAME.m3u` in the playlist directory.
 
     If a playlsit with name ``NAME`` already exists, the command fails with "Playlist already exists".
 
-.. _command_store:
+    ``MODE``
+        Optional argument [#since_0_xx_yy]_. One of `create`, `append`, or `replace`.
 
-:command:`store {NAME} {REPLACE}`
-    Saves the queue to an existing
-    `NAME.m3u` in the playlist directory.
-
-    If a playlsit with name ``NAME`` doesn\'t already exist, the command fails with "No such playlist".
+        `create`
+            The default. If a playlist with name ``NAME`` already exists, the command fails with "Playlist already exists".
```

Comment:
Actual error messages should not be documented, or else we can't ever rephrase them, because then clients will rely on the exact wording.